iina/OpenSubSubtitle.swift

```diff
@@ -186,13 +186,21 @@ class OpenSubSupport {
     }
   }
 
-  func hash(_ url: URL) -> Promise<FileInfo> {
+  func hash(_ url: URL, _ playerCore: PlayerCore) -> Promise<FileInfo> {
```

Comment:
This seems like the wrong point to bring in an entire PlayerCore, which handles video playback. Can the caller do this check instead?

Reply:
Fixed.